pkg/controllers/user/rbac/namespace_handler.go

```diff
@@ -118,6 +120,39 @@ func (n *nsLifecycle) ensurePRTBAddToNamespace(ns *v1.Namespace) (bool, error) {
 			return false, errors.Wrapf(err, "couldn't ensure binding %v in %v", prtb.Name, ns.Name)
 		}
 	}
+
+	var namespace string
+	if parts := strings.SplitN(projectID, ":", 2); len(parts) == 2 && len(parts[1]) > 0 {
+		namespace = parts[1]
+	} else {
+		return hasPRTBs, nil
+	}
+
+	rbs, err := n.m.rbLister.List(ns.Name, labels.Set(map[string]string{}).AsSelector())
```

Comment:
use `labels.Everything()` instead of `labels.Set(map[string]string{}).AsSelector()`